packages/rancher-logging/package.yaml

```diff
@@ -1,5 +1,5 @@
 url: oci://ghcr.io/kube-logging/helm-charts/logging-operator:4.4.0
-version: 103.1.1
+version: 103.1.2
```

Comment:
Can you add `-rc1` to the end? We've gone through some versioning issues during releases and the rc makes it all manageable.